### ISSUE TITLE
Crystal Convention for Methods

### ISSUE THREAD

**Issue by sdogruyol** [blocknotes/magickwand-crystal#1]
Hey @blocknotes 

Thanks a lot for this library. I saw that the method names aren't snake cased and were like `newMagickWand`.

In Crystal (also in Ruby) is to follow snake cases in method and variable names. So `newMagickWand` should be `new_magick_wand`.
 
WDYT?

**Comment by blocknotes** [comment]
Hello Serdar :)
I decided to use that format for 2 reasons:
- in my opinion a binding lib should be as near as possibile to the original lib because if the names match it's easier to find the original method and it's faster to adapt a C program to a Crystal one;
- I made some scripts (o I should call them programs for Crystal...) that check automatically the signatures, enums and structs - if the names don't match it gets tricky to keep the association.

I would change the method names for a wrapper (as I did in wkhtmltopdf-crystal). But currently I'm not thinking to a wrapper for magickwand, it's too big and it has too many options.
